io.github.piotrek_k._2dTaskBoard.yml

```diff
@@ -0,0 +1,62 @@
+app-id: io.github.piotrek_k._2dTaskBoard
+runtime: org.freedesktop.Platform
+runtime-version: '24.08'
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: '24.08'
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.node20
+command: run.sh
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --device=dri
+build-options:
+  append-path: /usr/lib/sdk/node20/bin
+  env:
+    NPM_CONFIG_LOGLEVEL: info
+modules:
+  - name: 2dTaskBoard
+    buildsystem: simple
+    build-options:
+      env:
+        XDG_CACHE_HOME: /run/build/2dTaskBoard/flatpak-node/cache
+        npm_config_cache: /run/build/2dTaskBoard/flatpak-node/npm-cache
+        npm_config_offline: 'true'
+    build-commands:
+      - |
+        cd desktop
+        npm install --offline
+      - |
+        cd web
+        npm install --offline
+        npm run build
+        mkdir -p ../desktop/dist/2dTaskBoard/
+        cp -r dist/* ../desktop/dist/2dTaskBoard/
+      - |
+        . ../flatpak-node/electron-builder-arch-args.sh
+        cd desktop
+        npm run dist -- $ELECTRON_BUILDER_ARCH_ARGS  --linux --dir
+      - |
+        cd desktop
+        cp -a release-builds/linux*unpacked /app/main
+      - install -Dm755 -t /app/bin/ ../run.sh
+      - | 
+        cd desktop/flathub
+        install -Dm644 io.github.piotrek_k._2dTaskBoard.desktop -t /app/share/applications
+        install -Dm644 io.github.piotrek_k._2dTaskBoard.metainfo.xml -t /app/share/metainfo
+        install -Dm644 io.github.piotrek_k._2dTaskBoard.svg -t /app/share/icons/hicolor/scalable/apps
+    subdir: main
+    sources:
+      - type: git
+        url: https://github.com/piotrek-k/2dTaskBoard.git
+        tag: v0.4.3
+        commit: 7d953f9dd7703141137815a11e2df61acda607a6
+        dest: main
+      - generated-sources-desktop.json
+      - generated-sources-web.json
+      - type: script
+        dest-filename: run.sh
+        commands:
+          - zypak-wrapper.sh /app/main/task-board-desktop "$@"
```

Comment:
desktop file is not using this wrapper so launching via that does not work. use `desktop-file-edit` to change it here.

Reply:
desktop file seems to be working now, here is what I've changed
https://github.com/piotrek-k/2dTaskBoard/commit/1b0fd4aeeb38fed06cf38c6588a4fc3f05d0ca01